defaults/main.yml

```diff
@@ -1,11 +1,32 @@
-# Dont change this value, it's value is referenced
+
+# Tarsnap key contents
+# Should look like
+# TARSNAP_KEY: |
+#      # START OF TARSNAP KEY FILE
+#      contents
+#      # END OF TARSNAP KEY FILE
+TARSNAP_KEY: null
 TARSNAP_KEY_REMOTE_LOCATION: /root/tarsnap.key
-TARSNAP_KEY_LOCAL_TEMPLATE: null
 
-TARSNAPPER_CONF: null
+# You'll need to upload these scripts by yourself
+TARSNAP_BACKUP_PRE_SCRIPT: null
+TARSNAP_BACKUP_POST_SCRIPT: null
+
```

Comment:
Ah, I had misread "So user of this role needs to upload pre and post scripts himself" - I thought you meant a manual upload of the file, when you meant having the role A upload it himself. 

We should still make sure that something fails in the ansible run if the tarsnap config isn't uploaded, to make sure client roles don't forget to do this.